### PR TITLE
[ACCESS] [REGEDIT] Improve pt-PT translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[CPL.ACCESS] Improve pt-PT translation
[CPL:ACCESS] Adjust object position for better appearance
[[REGEDIT] Update pt-PT translation and object adjustement

### CODE REVIEW

**Comment by zecarlos1957** [comment]
Before
After

**Comment by zecarlos1957** [comment]
Before
After

**Comment by zecarlos1957** [comment]
Before
FindBefore

After
FindAfter

File: base/applications/regedit/lang/pt-PT.rc [hunk 1]
```diff
@@ -512,24 +512,24 @@ END
...
             WS_TABSTOP, 14, 36, 35, 8
     CONTROL "&Valores", IDC_LOOKAT_VALUES, "Button", BS_AUTOCHECKBOX |
             WS_TABSTOP, 14, 48, 36, 8
     CONTROL "&Dados", IDC_LOOKAT_DATA, "Button", BS_AUTOCHECKBOX |
             WS_TABSTOP, 14, 60, 42, 8
-    CONTROL "Corresponder apenas &string inteira", IDC_MATCHSTRING, "Button", BS_AUTOCHECKBOX |
-            WS_TABSTOP, 83, 32, 94, 13
-    CONTROL "Cor&respondencia &", IDC_MATCHCASE, "Button", BS_AUTOCHECKBOX |
-            WS_TABSTOP, 83, 48, 90, 12
+    CONTROL "Corresponder apenas &cadeia completa", IDC_MATCHSTRING, "Button", BS_AUTOCHECKBOX |
```
Comment (HBelusca) [inline base/applications/regedit/lang/pt-PT.rc:529]: keyboard accelerator for "c" is already taken by "Chaves" button, see above.

File: dll/cpl/access/lang/pt-PT.rc [hunk 2]
```diff
@@ -136,14 +136,14 @@ CAPTION "Definições para ""Teclas lentas"""
 FONT 8, "MS Shell Dlg"
 BEGIN
     GROUPBOX "Atalho do teclado", -1, 6, 11, 234, 62
-    LTEXT "O atalho para ""Teclas lentas"" é:\nmanter a tecla SHIFT direita pressionada por oito segundos.",
+    LTEXT "O atalho para ""Teclas lentas"" é:\nManter premida a tecla SHIFT da direita  durante oito segundos.",
```
Comment (HBelusca) [inline dll/cpl/access/lang/pt-PT.rc:139]:
```suggestion
    LTEXT "O atalho para ""Teclas lentas"" é:\nManter premida a tecla SHIFT da direita durante oito segundos.",
```

**Review by HBelusca — CHANGES_REQUESTED** [review]

**Review by HBelusca — APPROVED** [review]

**Pull request merged**